sovrin_client/test/cli/helper.py

```diff
@@ -191,3 +198,45 @@ def new():
             with Looper(debug=False) as looper:
                 yield new()
     return _
+
+
+# marker class for regex pattern
+class P(str):
+    def match(self, other):
+        return re.match('^{}$'.format(self), other)
+
+
+def check_wallet(cli,
+                 totalLinks=None,
+                 totalAvailableClaims=None,
+                 totalSchemas=None,
+                 totalClaimsRcvd=None,
+                 within=None):
+    async def check():
+        assert (totalLinks is None or
+                totalLinks == len(cli.activeWallet._links))
+
+        tac = 0
+        for li in cli.activeWallet._links.values():
```

Comment:
One line (little faster) alternative, `tac = sum([len(li.availableClaims) for li in cli.activeWallet._links.values()])`